### PR TITLE
Dockerfile lints

### PR DESCRIPTION
**Pull request by brittanybelle**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 RUN apt install -y \
     curl \
     jq \
     less
 
 WORKDIR /app/
 
-CMD [ "jq" ]
\ No newline at end of file
+USER root
```
[the remark this answers is not in the document]
Comment (brittanybelle) [reply docker/Dockerfile:12]: @sonatype-lift ignoreall

File: alt/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM ubuntu
```
[the remark this answers is not in the document]
Comment (brittanybelle) [reply alt/Dockerfile:1]: some other comment

File: alt/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+USER root
+
+RUN apt-get install python
+
+RUN apt install -y \
+    curl \
+    jq \
+    less
+
+USER root
```
[the remark this answers is not in the document]
Comment (KirannKaur) [reply alt/Dockerfile:14]: <details>
<summary>Expand to find similar findings</summary>
| **File Path** | **Error Line Number** |
| ------------- | ------------- |
| jvm/apps/lift-backend/src/main/kotlin/com/sonatype/lift/badge/BadgeService.kt  | [58](https://www.sonatype.com)  |
| jvm/apps/…/lift/badge/BadgeService.kt  |[78](https://www.sonatype.com) |
| jvm/apps/lift-backend/src/test/groovy/com/sonatype/lift/badge/BadgeControllerIT.groovy  |[118](https://www.sonatype.com) |
| scripts/build/build.sh |[315](https://www.sonatype.com) |
</details>

[the remark this answers is not in the document]
Comment (KirannKaur) [reply alt/Dockerfile:14]: <details>

| **File Path** | **Error Line Number** |
| ------------- | ------------- |
| jvm/apps/lift-backend/src/main/kotlin/com/sonatype/lift/badge/BadgeService.kt  | [58](https://www.sonatype.com)  |
| jvm/apps/…/lift/badge/BadgeService.kt  |[78](https://www.sonatype.com) |
| jvm/apps/lift-backend/src/test/groovy/com/sonatype/lift/badge/BadgeControllerIT.groovy  |[118](https://www.sonatype.com) |
| scripts/build/build.sh |[315](https://www.sonatype.com) |
</details>

[the remark this answers is not in the document]
Comment (KirannKaur) [reply alt/Dockerfile:14]: | **File Path** | **Error Line Number** | 
| ------------- | ------------- | 
| jvm/apps/lift-backend/src/main/kotlin/com/sonatype/lift/badge/BadgeService.kt | [58](https://www.sonatype.com) | 
| jvm/apps/…/lift/badge/BadgeService.kt |[78](https://www.sonatype.com) | 
| jvm/apps/lift-backend/src/test/groovy/com/sonatype/lift/badge/BadgeControllerIT.groovy |[118](https://www.sonatype.com) | 
| scripts/build/build.sh |[315](https://www.sonatype.com) |

[the remark this answers is not in the document]
Comment (KirannKaur) [reply alt/Dockerfile:14]: <a href="#">test</a>

File: alt/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,20 @@
+FROM ubuntu
+
+RUN top
+
+USER root
+
+RUN apt-get install python
+
+RUN apt install -y \
```
[the remark this answers is not in the document]
Comment (KirannKaur) [reply alt/Dockerfile:9]: @sonatype-lift help

**Pull request closed without merging**